lib/dal/src/management/mod.rs

```diff
@@ -485,7 +493,11 @@ impl<'a> ManagementOperator<'a> {
             auto_geometry
         };
 
-        Ok((component.id(), geometry))
+        Ok(CreatedComponent {
+            component,
+            geometry,
```

Comment:
Doesn't this need a viewId? (i.e. all geometries are related to a view)

Reply:
This is the relative geometry within the current view, which will get transformed into the view geo when we call `set_geometry`. Everything is happening within the view passed in (right now that is only the default view)